packages/driver/src/adapter.ts

```diff
@@ -1,20 +1,23 @@
-import * as crypto from "crypto";
-import { promises as fs } from "fs";
-import * as net from "net";
-import * as os from "os";
-import * as path from "path";
-import * as tls from "tls";
+import * as crypto from "node:crypto";
```

Comment:
This file probably shouldn't exist anymore, as there's nothing it "adapts" -- there's just one thing here as far as I can see.

Reply:
We use functions from this file: walk, exists, readFileUtf8, input, hasFSReadPermission, hashSHA1toHex. I renamed it to `systemUtils.ts`. If you think `fsUtils` or some other name is better I can update. I removed exports from this file that can be directly imported from node at other places.

Reply:
good stuff